ADAL/src/ADRequestParameters.h

```diff
@@ -28,32 +28,22 @@
 @class MSIDAccountIdentifier;
 
 @interface ADRequestParameters : NSObject <MSIDRequestContext>
-{
-    NSString *_authority;
-    NSString *_resource;
-    NSString *_clientId;
-    NSString *_redirectUri;
-    NSString *_scopesString;
-    ADUserIdentifier *_identifier;
-    BOOL _extendedLifetime;
-    BOOL _forceRefresh;
-    NSUUID *_correlationId;
-    NSString *_telemetryRequestId;
-}
 
-@property (retain, nonatomic) NSString* authority;
-@property (retain, nonatomic) NSString* cloudAuthority;
-@property (retain, nonatomic) NSString* resource;
-@property (retain, nonatomic) NSString* clientId;
-@property (retain, nonatomic) NSString* redirectUri;
-@property (retain, nonatomic) NSString* scopesString;
-@property (retain, nonatomic) ADUserIdentifier* identifier;
+@property (retain, nonatomic) NSString *authority;
+@property (retain, nonatomic) NSString *cloudAuthority;
+@property (retain, nonatomic) NSString *resource;
+@property (retain, nonatomic) NSString *clientId;
+@property (retain, nonatomic) NSString *redirectUri;
+@property (retain, nonatomic) NSString *scopesString;
+@property (retain, nonatomic) ADUserIdentifier *identifier;
+@property (retain, nonatomic) NSString *claims;
```

Comment:
Should we add claims and extra query parameters also into the initializer?

Reply:
We don't use the initializer in the code, I will actually remove the initializer.